host_core/lib/host_core/control_interface/acl.ex

```diff
@@ -4,30 +4,44 @@ defmodule HostCore.ControlInterface.ACL do
   def all_actors() do
     HostCore.Actors.ActorSupervisor.all_actors()
     |> Enum.flat_map(fn {id, pids} ->
+      name = get_name(id)
       revision = get_revision(id)
 
+      instances =
+        pids
+        |> Enum.map(fn pid ->
+          %{
+            annotations: %{},
```

Comment:
Should this be getting annotations from somewhere?

Reply:
I see support for annotations on the control interface commands for starting an actor but we don't actually use them ATM.

Reply:
Annotations aren't fully supported yet.